scripts/migrate-vfolder-structure.py

```diff
@@ -8,33 +8,26 @@ def main():
     print(input_path, output_path)
     subfolders = [x for x in input_path.iterdir() if x.is_dir()]
 
-    print("Subfolders ", subfolders)
-
-    for folder in subfolders:
-        folder = str(folder)
-        folder = folder.split("/")[-1]
-        header_dir = Path(folder[0:2])
-        second_lvl_dir = Path(folder[2:4])
-        rest_dir = Path(folder[4:])
-
-        root = Path(input_path / Path(folder))
-        list_of_files = [x for x in root.iterdir() if x.is_file()]
-        print("list of files ", list_of_files)
-
-        try:
-            if not Path.is_dir(output_path / header_dir):
-                Path.mkdir(output_path / header_dir)
-            if not Path.is_dir(output_path / header_dir / second_lvl_dir):
-                Path.mkdir(output_path / header_dir / second_lvl_dir)
-            if not Path.is_dir(output_path / header_dir / second_lvl_dir / rest_dir):
-                Path.mkdir(output_path / header_dir / second_lvl_dir / rest_dir)
+    print("Number of folders ", len(subfolders))
+    try:
+
+        for folder in subfolders:
+            folder = str(folder)
+            folder = folder.split("/")[-1]
+            header_dir = Path(folder[0:2])
+            second_lvl_dir = Path(folder[2:4])
+            rest_dir = Path(folder[4:])
+
+            Path(output_path / header_dir).mkdir(parents=True, exist_ok=True)
+            Path(output_path / header_dir / second_lvl_dir).mkdir(parents=True, exist_ok=True)
+            Path(output_path / header_dir / second_lvl_dir / rest_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
Still this could be reduced to one line, because the parents of the last line's Path includes/implies the first line and the second line.